r/DESCRIPTION

```diff
@@ -59,6 +59,7 @@ Suggests:
     testthat (>= 3.1.0),
     tibble,
     withr
+LinkingTo: cpp11 (>= 0.4.2)
```

Comment:
This should be the relevant change we need, cf. https://github.com/apache/arrow/runs/4550863089?check_suite_focus=true#step:16:224

Reply:
Yes, this prevents the building errors above, but then we get `Package required and available but unsuitable version: 'cpp11'` when running `R CMD CHECK`

https://github.com/apache/arrow/runs/4741782441?check_suite_focus=true#step:17:537